docs/docs/overview/docker-deploy.md

```diff
@@ -95,6 +95,18 @@ When using the IFrame embed to attach the notification center rather than the Re
 
 See [Use your own backend and socket URL](https://docs.novu.co/notification-center/react-components#use-your-own-backend-and-socket-url).
 
+### Caching
+
+We are introducing the first stage of caching in our system to improve performance and efficiency. Caching is turned off by default, but can easily be activated by setting the following environment variables:
```

Comment:
should we make the 'Caching is turned off by default' part more prominent somehow?

Reply:
I think we could specify that we are currently feature-flagging our caching implementation.
If a user has a use-case and wants to opt-in to cache they can turn it on by setting the following environment variables.

Reply:
Hey @Cliftonz not sure if I got it there needed to be changes or not :), if there needed to be a change can you point me in the direction where you think we need, to add, or update some parts?